project-communications/newsletter/newsletter-012921.md

```diff
@@ -0,0 +1,104 @@
+
+>"I think something interesting about my profile, which I think is becoming more and more common now, is I've lived life both sides of the river. I've lived for 46 years as a neurotypical person, and I've only lived the past year and a half knowing that I was autistic. Getting a diagnosis was life-changing for me. My motivation wasn't because I needed extra support, it was more because of curiosity, and because of my daughter. My diagnosis came as a positive thing, as a quest of curiosoty and self-knowledge, rather than being because I was experiencing problems in my life. So that's actually quite an unusual thing. I wasn't "diagnosed" in that pathology paradigm of something being wrong. So for me, I think that's quite an interesting angle, that my diagnosis was about something more, it was about my strengths - but also so that I could locate some of the challenges that I'd previously experienced in my neurotypical life.""
+
+>"For me it's just a different way of being in the world, which describes my relationship to the world and people. I always grew up feeling different, and not really understanding why. I had different feelings about things, different reactions to things, and I felt like a refugee actually - which is quite interesting given my work. I felt dislocated and alienated. So when I got my diagnosis I could make sense of why I felt and thought differently. It was validating. A lot of the things I would criticise myself for, like that I couldn't understand things in the same way other people did, or that I couldn't think about things in the same way - can lead to really negative feelings about yourself, which can become a self-perpetuating cycle."" 
+
+>"Some autistic people have quite switched off experiences, but for me, I experience life in glorious technicolour. Don't get me wrong, I'm not seeing things dancing - it's rooted in reality, but I see and feel things very intensely. My particular profile is I'm hyper-sensitive and hyper-emotional. So an example of that would be that if I saw a suffering animal or a dead animal, I would experience pain. My upset will manifest in internal pain, and it will literally be like experiencing high emotions and pain for a couple of days. You're carrying this trauma of this animal with you. So that's an example of how I might experience an event - or something that's difficult or sad. Being autistic can make me different to people, so it can disconnect me - but also, for me, I feel really proud to be autistic. It's given me strength and it's given me power, and being autistic explains my success in life. I'm autistic with attention deficiencies, so for example the impulsivity associated with ADHD can lead some peoplee into negative, painful experiences. That impulsivity led me into something really positive - into risk-taking, because I thought, 'yeah, I'm going to set my own business up'."" 
```

Comment:
```suggestion
> "Some autistic people have quite switched off experiences, but for me, I experience life in glorious technicolour. Don't get me wrong, I'm not seeing things dancing - it's rooted in reality, but I see and feel things very intensely. My particular profile is I'm hyper-sensitive and hyper-emotional. So an example of that would be that if I saw a suffering animal or a dead animal, I would experience pain. My upset will manifest in internal pain, and it will literally be like experiencing high emotions and pain for a couple of days. You're carrying this trauma of this animal with you. So that's an example of how I might experience an event - or something that's difficult or sad. Being autistic can make me different to people, so it can disconnect me - but also, for me, I feel really proud to be autistic. It's given me strength and it's given me power, and being autistic explains my success in life. I'm autistic with attention deficiencies, so for example the impulsivity associated with ADHD can lead some people into negative, painful experiences. That impulsivity led me into something really positive - into risk-taking, because I thought, 'yeah, I'm going to set my own business up'."" 
```